content-src/asrouter/templates/CFR/templates/ExtensionDoorhanger.schema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "title": "ExtensionDoorhanger",
+  "description": "A template with a heading, addon icon, title and description. No markup allowed.",
+  "version": "1.0.0",
+  "type": "object",
+  "definitions": {
+    "plainText": {
+      "description": "Plain text (no HTML allowed)",
+      "type": "string"
+    },
+    "linkUrl": {
+      "description": "Target for links or buttons",
+      "type": "string",
+      "format": "uri"
+    }
+  },
+  "properties": {
+    "heading": {
+      "type": "object",
+      "properties": {
+        "title": {
+          "allOf": [
+            {"$ref": "#/definitions/plainText"},
+            {"description": "Tooltip heading describing it's purpose."}
+          ]
+        },
+        "url": {
+          "allOf": [
+            {"$ref": "#/definitions/linkUrl"},
+            {"description": "Link to support page with explanation."}
+          ]
+        }
+      }
+    },
+    "addon": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "allOf": [
+            {"$ref": "#/definitions/plainText"},
+            {"description": "Addon name."}
+          ]
+        },
+        "author": {
+          "allOf": [
+            {"$ref": "#/definitions/plainText"},
+            {"description": "Addon author."}
+          ]
+        },
+        "icon": {
+          "allOf": [
+            {"$ref": "#/definitions/linkUrl"},
+            {"description": "Addon icon."}
+          ]
+        }
+      }
+    },
+    "content": {
+      "type": "object",
+      "properties": {
+        "text": {
+          "allOf": [
+            {"$ref": "#/definitions/plainText"},
+            {"description": "Description of the addon."}
+          ]
+        },
+        "url": {
```

Comment:
This specifically the AMO link, so maybe add this to the addon section and call it `amo_url` or something